src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -16,6 +18,8 @@ internal class ConsoleGui : IDisposable
         private const string CANCEL_TEXT = "Are you sure you want to cancel?\nNothing will be emitted to the pipeline.";
         private const string CLOSE_TEXT = "Are you sure you want to close?";
         private bool _cancelled;
+        private List<string> _itemList;
+        private ListView _list;
```

Comment:
I'd prefer if this was called `_listView` so I don't have to look at the type to know what it is but I won't block on this.

Reply:
ok